d4s2_api_v2/api.py

```diff
@@ -10,12 +10,14 @@
 from d4s2_api_v2.serializers import DDSUserSerializer, DDSProjectSerializer, DDSProjectTransferSerializer, \
     UserSerializer, S3EndpointSerializer, S3UserSerializer, S3BucketSerializer, S3DeliverySerializer, \
     DDSProjectPermissionSerializer, DDSDeliveryPreviewSerializer
-from d4s2_api.models import DDSDelivery, S3Endpoint, S3User, S3UserTypes, S3Bucket, S3Delivery, UserEmailTemplateSet
+from d4s2_api.models import DDSDelivery, S3Endpoint, S3User, S3UserTypes, S3Bucket, S3Delivery
 from d4s2_api_v1.api import AlreadyNotifiedException, get_force_param, build_accept_url, DeliveryViewSet, \
-    EMAIL_TEMPLATES_NOT_SETUP_MSG
+    get_email_template_for_request, populate_email_template_in_request, prevent_email_template_set_in_request, \
```

Comment:
Importing this many functions from v1 for this change seems like a code smell. As noted earlier, I think some of it can be handled by the serializer, but the rest of the logic is probably better encapsulated in a Mixin.